test/button_test.js

```diff
@@ -10,6 +10,100 @@ const Context = firefox.Context;
 // TODO create new profile per test?
 // then we can test with a clean profile every time
 
+async function urlBarTest(driver) {
```

Comment:
After discussion, I decided to move most of the tests back into the `describe()` if they have no reason to be repeated.